### PR TITLE
vonk: Release notes 2.0.0 final

### PR DESCRIPTION
**Pull request by cknaap**

### CODE REVIEW

File: vonk/releasenotes/releasenotes.rst [hunk 1]
```diff
@@ -10,8 +10,61 @@ Release notes Vonk
...
+#. Vonk supports Custom Resources. See :ref:`feature_customresources`.
+#. Operation :ref:`feature_meta` is now supported, to quickly get the tags, security labels and profiles of a resource.
+#. /metadata, retrieving the CapabilityStatement performs a lot better (just the initial call for a specific Accept-Type takes a bit longer).
+#. Validation can be controlled more detailed. Choose the strictness of parsing independent of the level of validation. With this, the settings section 'Validation' has also changed. See :ref:`feature_prevalidation`. 
+
+Plugin and Facade API
+^^^^^^^^^^^^^^^^^^^^^
+
+#. We upgraded the embedded Fhir.Net API to version 1.2, see its :ref:`release notes <api_releasenotes_1.2.0>`.
+#. Together with the upgrade to .NET Core 2.2, several libraries were updated as well. Most notably Microsoft.EntityFrameworkCore.*, to 2.0.1.
```
Comment (alexzautke) [inline vonk/releasenotes/releasenotes.rst]: Shouldn't this be 2.2.*?

Comment (cknaap) [reply vonk/releasenotes/releasenotes.rst]: I see 2.0.1 in Vonk.Repository.Sql.csproj....

Comment (alexzautke) [reply vonk/releasenotes/releasenotes.rst]: https://git.furore.com/projects/FHIR/repos/vonk/browse/src/Vonk.Repository.Sql/Vonk.Repository.Sql.csproj

Are we taking about the same file? :sweat_smile:

Comment (abelenthoven) [reply vonk/releasenotes/releasenotes.rst]: it is indeed 2.2.3

Comment (cknaap) [reply vonk/releasenotes/releasenotes.rst]: Corrected.

**Pull request merged**